web/src/todo_widget.ts

```diff
@@ -101,8 +91,13 @@ export class TaskData {
                 return undefined;
             },
 
-            inbound: (sender_id, data) => {
+            inbound: (sender_id: number, raw_data: unknown): void => {
                 // Only the message author can edit questions.
+                const new_task_title_inbound_data = z.object({
+                    type: z.literal("new_task_list_title"),
+                    title: z.unknown(),
```

Comment:
Why we are parsing the title as `unknown` here and then anyways checking if it's a string or not later?

Reply:
Yeah, I agree. I changed the `z.unknown()` to `z.string()` and `safeParsed` the incoming data in the newer version. Instead of those `typeof` checks we should rely on `zod` -- that is the whole point of it. Thanks, done!